modules/nf-core/fastcov/tests/main.nf.test

```diff
@@ -0,0 +1,60 @@
+nextflow_process {
+
+    name "Test Process FASTCOV"
+    script "../main.nf"
+    process "FASTCOV"
+    config "./nextflow.config"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "fastcov"
+
+    test("fastcov single input bam") {
+
+        when {
+            process {
+                """                
+                input[0] = [
+                    [ id:'test' ], // meta map
+                    [ file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam', checkIfExists: true) ],
+                    file(params.modules_testdata_base_path + 'genomics/sarscov2/illumina/bam/test.paired_end.sorted.bam.bai', checkIfExists: true)
+                    ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
```

Comment:
Can you test the output of your test run a bit more? For example assert if the file is always a png (if that should be that way) or if the file name stays consitent?